src/LanguageServer.Common/Utilities/MSBuildHelper.cs

```diff
@@ -74,9 +74,6 @@ public static class MSBuildHelper
         /// </param>
         public static void DiscoverMSBuildEngine(string baseDirectory = null, ILogger logger = null)
         {
-            if (MSBuildLocator.IsRegistered)
-                MSBuildLocator.Unregister();
```

Comment:
In the latest version `MSBuildLocator.Unregister()` [is just a stub for backward compatibility](https://github.com/microsoft/MSBuildLocator/blob/0bd7d2689d9e0e03a4b0e5bcdccf053f61887f90/src/MSBuildLocator/MSBuildLocator.cs#L286-L292). No need to call it anymore